commands/login.js

```diff
@@ -24,9 +24,7 @@ async function login (context, heroku) {
   try {
     await dockerLogin(registry, password, context.flags.verbose)
   } catch (err) {
-    cli.error(`Error: docker login exited with ${err}`)
-    cli.hush(err.stack || err)
```

Comment:
What did this do?

Reply:
It appears it shows the javascript stack (for a docker error which seems weird).
The main reason I removed it is that `cli.exit()` exits the tests too, preventing us from properly testing.

`cli.error()` takes a second argument which is the exit code. But `cli.hush()` isn't documented.